compose/service.py

```diff
@@ -365,6 +365,13 @@ def convergence_plan(self, strategy=ConvergenceStrategy.changed):
 
         return ConvergencePlan('noop', containers)
 
+    def get_config_hash(self):
```

Comment:
We're not writing Java here - let's just access the property directly :slightly_smiling_face: